website_event_webinars/.gitignore

```diff
@@ -0,0 +1,352 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Remove file

Reply:
Hi emagdalenaC2i. 
Just to confirm that this module compliments the website_event_require_login, does not replace it.
website_event_require_login forces login before allowing registration for the event.
My module website_event_webinars goes further to also FORCE that when registering for a ticket for an event, it can only be registered under the logged-in name and email address.
Without it, you may free-type any other person's name and email into the ticket registration box. For "invite only" webinar events, you may need to vet people beforehand and therefore not allow anonymous ticket submissions.
Hope that helps.
Chris

Reply:
Ok, thanks